### ISSUE TITLE
Text zoom on Firefox appears to break areas of the site

### ISSUE THREAD

**Issue by joelanman** [alphagov/govuk-design-system#2028]
## Description of the issue
When text zoom is set to 200% in Firefox, areas of the site appear to break:
![screenshot showing a link with the underline through the text](https://user-images.githubusercontent.com/1132904/150353997-f27aa859-ac6f-45b3-a32e-fe436e58d1d9.png)
![screenshot showing the navigation links obscured by the blue panel below](https://user-images.githubusercontent.com/1132904/150354022-3f7e4cde-36a7-4709-ab23-07a1e238d3c8.png)

## Steps to reproduce the issue
Browse the site at 200% text zoom in Firefox

## Actual vs expected behaviour
No content should be obscured or broken with text zoom, it does appear to be in Firefox at 200% zoom

## Environment (where applicable)

- Operating system: MacOS
- Browser: Firefox

**Comment by joelanman** [comment]
I have attempted to recreate in Chrome and Safari, but have not been able to, however I don't know the text zoom controls very well
